src/app/shared/export-to-justifi-modal/export-to-justifi-modal.component.html

```diff
@@ -0,0 +1,190 @@
+<div bsModal #exportToJustifiModal="bs-modal" class="modal fade" tabindex="-1" role="dialog"
+    aria-labelledby="exportToJustifiModal" aria-hidden="true" [config]="{backdrop: 'static'}">
+    <div class="modal-dialog modal-md">
+        <div class="modal-content">
+            <div class="modal-header p-2">
+                <h5 class="mt-auto">Export Content to JUSTIFI Template</h5>
+                <button class="close p-0 m-0" aria-label="Close" (click)="hideModal()">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body p-2">
+                <p class="mb-0">
```

Comment:
1. Second sentence of this has typo or needs grammatical touch up "you can the... has compiled"
2. a nit, but would be great if we can make the modal 50% or even 75% page width. Better readability, more in line with other MEASUR modals

Reply:
updated for responsive modal width on large screens and fixed typo